test/testsupport/ItSpec.scala

```diff
@@ -56,7 +59,9 @@ class ItSpec
   protected val overrideConfig: Map[String, Any] = Map.empty
 
   protected lazy val configMap: Map[String, Any] = Map[String, Any](
+    "mongodb.uri" -> "mongodb://localhost:27017/self-service-time-to-pay-frontend-tests",
```

Comment:
I've made the tests use a separate database then the one used by the actual local instance - I found the collections I had were bloated with a load of journeys which I hadn't actually gone through locally myself. I made the test clean up the data it creates as well down below.